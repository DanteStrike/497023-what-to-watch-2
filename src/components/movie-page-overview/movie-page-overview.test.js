import React from "react";
import renderer from "react-test-renderer";

import MoviePageOverview from "./movie-page-overview.jsx";


it(`Render correctly MoviePageOverview component`, () => {
  const component = renderer
    .create(
        <MoviePageOverview
          rating={10}
          scoresCount={5}
          description={`any`}
          director={`director`}
          starring={[`star`, `star`]}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
