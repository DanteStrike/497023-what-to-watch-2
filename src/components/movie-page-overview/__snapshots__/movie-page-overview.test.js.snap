// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly MoviePageOverview component 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      10
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Awesome
      </span>
      <span
        className="movie-rating__count"
      >
        5
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      any
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        star, 
        star
         and other
      </strong>
    </p>
  </div>,
]
`;
